app/helpers/react_components/track/approach.rb

```diff
@@ -0,0 +1,9 @@
+module ReactComponents
+  module Track
+    class Approach < ReactComponent
+      def to_s
+        super("track-approach", {})
+      end
```

Comment:
This file is unused atm. I created for the Approach show, then ended up writing all of it in haml. Maybe leave it for now in case we need a React component. or delete it immediately?

Reply:
If you think the HAML version will suffice, you can remove this